core/src/test/fixtures_postgresql/column_types/Sample.s

```diff
@@ -45,6 +45,12 @@ CREATE TABLE all_types(
 
   some_timestamptz_shorthand TIMESTAMPTZ,
 
+  some_always_generated_identity BIGINT GENERATED ALWAYS AS IDENTITY,
+  some_default_generated_identity BIGINT GENERATED BY DEFAULT AS IDENTITY,
+
+  height_in_cm NUMERIC,
+  generated_height_in_cm_stored NUMERIC GENERATED ALWAYS AS ( height_in_cm / 2.54 ) STORED,
```

Comment:
This PR just covers simple grammar support. I unsure what it would take to perform validation on `( height_in_cm / 2.54 )`.

Reply:
it should be already validated, you can try adding one that fails (with an unknown column expression) to verify that

Reply:
yup, it works!